RELEASE.md

```diff
@@ -85,6 +86,16 @@ Builder branch cut should be performed at the same time as Pytorch core branch c
 DRY_RUN=disabled GIT_BRANCH_TO_CUT_FROM=main RELEASE_VERSION=1.11 scripts/release/cut-release-branch.sh
 ```
 
+### RC cut health validation
```

Comment:
Shouldn't this section be split in two or moved after "Drafting RCs (Release Candidates) for PyTorch and domain libraries" section? 
```suggestion
### Release Candidate health validation
```